exporter/collector/logs.go

```diff
@@ -434,12 +434,15 @@ func (l logMapper) logToSplitEntries(
 	}
 	entry.Severity = severityMapping[severityNumber]
 
+	// log.Body().AsString() can be expensive, and we use it several times below this, so
+	// do it once and save that as a variable.
+	logBodyString := logRecord.Body().AsString()
```

Comment:
It looks like we could avoid this call altogether in the map case, I'm wondering if it's worth optimizing for this.

Is a map-typed body our recommended way of doing structured logging in OTel -> collector?

Reply:
moved it to after we handle maps and bytes.  This means we can send empty bytes as json, but that is probably fine

Reply:
I'm not sure about the second question.  I suspect maps are easiest, but i'm not sure what structured json would parse into later if you parsed a structured log.

Reply:
The question was mostly aiming to understand if its worth optimizing. The change LGTM so resolving